src/composables/ViewRoutingSlip/usePaymentInformation.ts

```diff
@@ -0,0 +1,49 @@
+import { ComputedRef, computed, ref, watch } from '@vue/composition-api'
+
+import { PaymentMethods } from '@/util/constants'
+import { RoutingSlip } from '@/models/RoutingSlip'
+import { createNamespacedHelpers } from 'vuex-composition-helpers'
+
+const routingSlipModule = createNamespacedHelpers('routingSlip') // specific module name
+const { useState, useMutations } = routingSlipModule
+
+// Composable function to inject Props, options and values to PaymentInformation component
+export default function usePaymentInformation () {
+  // UI control variables
+  const isExpanded = ref<boolean>(false)
+
+  // vuex action and state
+  const { routingSlip } = useState(['routingSlip'])
+  const { setChequePayment, setCashPayment } = useMutations(['setChequePayment', 'setCashPayment'])
+
+  // As per current business rule, a routingslip has one-to-one relation with payment method (Cash/Cheque)
+  // Therefore, we can determine the payment method of the current routingslip from the first payment record
+  const isPaymentCheque = computed(() => {
+    const payments = (routingSlip.value as RoutingSlip)?.payments
+    // to prevent lazy load
+    return payments && payments[0].paymentMethod === PaymentMethods.CHEQUE
+  })
+
+  function addAdditionalFunds (): void {
+  }
```

Comment:
unused function?

Reply:
For add additionalfunds. for future. But I can remove :)